eng/pipelines/templates/common.yml

```
@@ -190,6 +190,26 @@ extends:
           jobs:
           - template: /eng/pipelines/templates/jobs/release.yml
 
+        - stage: Docker
+          displayName: 'Docker Build and Publish'
+          dependsOn:
+          - Initialize
+          - SignAndPack
+          pool:
+            name: $(LINUXPOOL)
+            image: $(LINUXVMIMAGE)
+            os: linux
+          variables:
+          - template: /eng/pipelines/templates/variables/image.yml
+          - template: /eng/pipelines/templates/variables/globals.yml
+          - name: Version
+            value: $[ stageDependencies.Initialize.Initialize.outputs['Version.Version'] ]
+          jobs:
+          - template: /eng/pipelines/templates/jobs/docker.yml
```

Could this be added to the existing Release stage?

```suggestion
          - template: /eng/pipelines/templates/jobs/release.yml
          - template: /eng/pipelines/templates/jobs/docker.yml
```